dts/arm/nordic/nrf54h20_cpurad.dtsi

```diff
@@ -59,21 +59,37 @@ wdt011: &cpurad_wdt011 {};
 };
 
 &dppic130 {
-	owned-channels = <0>;
-	sink-channels = <0>;
-	nonsecure-channels = <0>;
+	owned-channels = <0 2 3>;
+	sink-channels = <0 2>;
+	source-channels = <3>;
+	nonsecure-channels = <0 2 3>;
 	status = "okay";
 };
 
 &dppic132 {
-	owned-channels = <0>;
-	source-channels = <0>;
-	nonsecure-channels = <0>;
+	owned-channels = <0 2 3>;
+	sink-channels = <3>;
+	source-channels = <0 2>;
+	nonsecure-channels = <0 2 3>;
 	status = "okay";
 };
 
 &ipct130 {
-	owned-channels = <0>;
-	source-channel-links = <0 3 0>;
 	status = "okay";
+	owned-channels = <0 2>;
+	sink-channel-links = <0 3 0>;
+	source-channel-links = <0 3 0
+					2 3 2>;
```

Comment:
`source-channel-links 2 3 2` is used by nrf-802154, right? It that case, would it be possible to set `sink-channel-links = <2 3 2>` as well?

Reply:
Yes, it is used. But why setting as 'sink'? The <2 3 2> is part of a link that triggers a radio task with GRTC.CC. I don't understand why setting it additionally as 'sink', while the connection chain goes from ipct130 to cpurad_ipct

Reply:
IPCT channels are always allocated in pairs, the sink and the source. I think that since nrf-802154 uses one sink and source, it would be better to use the same IPCT pair for nrf-802154 purposes.

Reply:
Makes sense, thanks

Reply:
I think the same applies to cpurad_ipct - I will make the analogic change there.